pkg/quota/admission/resourcequota/admission.go

```diff
@@ -54,7 +54,7 @@ func (a *originQuotaAdmission) Admit(as admission.Attributes) error {
 }
 
 func (a *originQuotaAdmission) SetOpenshiftClient(osClient osclient.Interface) {
-	registry := quota.NewRegistry(osClient)
+	registry := quota.NewRegistry(osClient, true)
```

Comment:
@pweil- I am not sure I follow.  The `quota.NewRegistry` code is completely unique to OpenShift.  The Kubernetes expectation is that you just return a `kquota.Registry` that knows how to evaluate use for OpenShift specific API types.  Maybe this comment will become clearer to me when I look at the rest of this PR.